wolfcrypt/src/sha256.c

```diff
@@ -516,7 +516,6 @@ static int InitSha256(wc_Sha256* sha256)
     }
     int wc_InitSha256_ex(wc_Sha256* sha256, void* heap, int devId)
     {
-        WOLFSSL_ENTER("wc_InitSha256_ex");
```

Comment:
Why is this being removed?  If it needs to be moved down a line, move it down a line.

Reply:
This message was useful at the beginning of acceleration implementation. After that, when enabling DEBUG_WOLFSSL, there were messages by this too many. By too many messages caused difficult to find a message what we really wanted to see, such as messages at handshake which users would also wanted to see. Therefore, I decided to remove it.